tests/metadata/test_rtc_gamma.py

```diff
@@ -43,10 +43,12 @@ def test_rtc_gamma_product(rtc_product_dir):
         rtc_product_dir / 'S1A_IW_20150621T120220_DVP_RTC10_G_saufem_F8E2_VH.tif.xml',
     ]
     for output_file in output_file_list:
+        assert output_file is not None
         assert output_file.exists()
 
 
 def test_create_dem_xml(rtc_product_dir):
+    dem_name: str | None = None
```

Comment:
You actually don't need to assign a value, it can just be a standalone type annotation.
```suggestion
    dem_name: str | None
```